From a3f025fc936c421d2494858219e71311783570fb Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Tue, 13 Jun 2017 09:52:54 +0100
Subject: [PATCH 7/9] crypto: cavium - fix spelling mistake "Revsion" ->
 "Revision"
References: FATE#323034
Git-commit: 7c742df55035ce566b2ca7d4125d1202d41a99f3
Patch-mainline: v4.13-rc1

Trivial fix to spelling mistake in seq_printf message

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/crypto/cavium/nitrox/nitrox_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/cavium/nitrox/nitrox_main.c b/drivers/crypto/cavium/nitrox/nitrox_main.c
index eee6fb501580..ae44a464cd2d 100644
--- a/drivers/crypto/cavium/nitrox/nitrox_main.c
+++ b/drivers/crypto/cavium/nitrox/nitrox_main.c
@@ -399,7 +399,7 @@ static int nitrox_show(struct seq_file *s, void *v)
 	struct nitrox_device *ndev = s->private;
 
 	seq_printf(s, "NITROX-5 [idx: %d]\n", ndev->idx);
-	seq_printf(s, "  Revsion ID: 0x%0x\n", ndev->hw.revision_id);
+	seq_printf(s, "  Revision ID: 0x%0x\n", ndev->hw.revision_id);
 	seq_printf(s, "  Cores [AE: %u  SE: %u]\n",
 		   ndev->hw.ae_cores, ndev->hw.se_cores);
 	seq_printf(s, "  Number of Queues: %u\n", ndev->nr_queues);
-- 
2.12.3

